CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+# Code of Conduct
+
+This code of conduct applies to the maintainers and contributors alike.
+
+## Dealing with issues and support requests
+
+_We wish to add a specific section on dealing with issues opened against the
```

Comment:
This reads like a TODO item. Perhaps a better approach would be to create an issue that describes the desired work ... once the pull request is accepted. So maybe that's something of a chicken-and-egg problem.